app/assets/javascripts/common_chart_options.js

```diff
@@ -221,32 +207,38 @@ function barColumnLine(chartData, highchartsChart, seriesData, chartConfig) {
     } else {
       highchartsChart.update({ plotOptions: { column: { tooltip: { headerFormat: '<b>{series.name}</b><br>', pointFormat: orderedPointFormat(yAxisLabel)}}}});
     }
+  }
 
-    if (y2AxisLabel) {
-
-      var axisTitle;
-      var pointFormat;
-      var max;
-
-      console.log('Y2 axis label' + y2AxisLabel);
-      colour = '#232b49';
-      if (y2AxisLabel == 'Temperature') {
-        axisTitle = '°C';
-        pointFormat = '{point.y:.2f} °C';
-      } else if (y2AxisLabel == 'Degree Days') {
-        axisTitle = 'Degree days';
-        pointFormat = '{point.y:.2f} Degree days';
-      } else if (isAStringAndStartsWith(y2AxisLabel, 'Carbon Intensity')) {
-        axisTitle = 'kg/kWh';
-        pointFormat = '{point.y:.2f} kg/kWh';
-        max = 0.5;
-      } else if (y2AxisLabel == 'Solar Irradiance') {
-        axisTitle = 'Brightness of sunshine W/m2';
-        pointFormat = '{point.y:.2f} W/m2';
-      }
-      highchartsChart.addAxis({ title: { text: axisTitle }, stackLabels: { style: { fontWeight: 'bold',  color: colour }}, opposite: true, max: max});
-      highchartsChart.update({ plotOptions: { line: { tooltip: { headerFormat: '<b>{point.key}</b><br>',  pointFormat: pointFormat }}}});
+  // Handle Y2 axis
```

Comment:
This was being handled in two separate places already, so pulled all together, hopefully haven't missed out...